recipes/web3/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "web3" %}
+{% set version = "5.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f378a773886122d2126d7ba81e9f5b6d2e7b4def7e88543699bf85a951835942
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-markdown
+  run:
+    - python <3.8
+    - eth-abi >=2.0.0b6,<3.0.0
+    - eth-account >=0.5.2,<0.6.0
+    - eth-hash >=0.2.0,<1.0.0
+    - eth-typing >=2.0.0,<3.0.0
+    - eth-utils >=1.8.4,<2.0.0
+    - hexbytes >=0.1.0,<1.0.0
+    - ipfshttpclient >=0.4.13,<1
+    - jsonschema >=3.2.0,<4.0.0
+    - lru-dict >=1.1.6,<2.0.0
+    - protobuf >=3.10.0,<4
+    # - pypiwin32 >=223  # [win]
+    - requests >=2.16.0,<3.0.0
+    - typing-extensions >=3.7.4.1,<4  # [py<38]
+    - websockets >=8.1.0,<9.0.0
+
+test:
+  imports:
+    - web3
+    - ens
+    - ethpm
+  commands:
+    - pip check
```

Comment:
This is not a common check of correctness in conda. 

Can you explain what you are attempted to verify here?

Reply:
grayskull did this. I will remove it and maybe add the proper tests.